packages/pulp/pulp.spec

```diff
@@ -289,6 +290,10 @@ ln -s %{_var}/lib/pulp/content %{buildroot}/%{_var}/www/pulp/nodes
 # Tools
 cp server/bin/* %{buildroot}/%{_bindir}
 
+# pulp-maintenance
+install -d -m0755 %{buildroot}%{_sysconfdir}/cron.monthly
+install -m 755 %{SOURCE1} %{buildroot}/%{_sysconfdir}/cron.monthly/pulp-maintenance
```

Comment:
Yes weekly is what we confirmed on IRC w/ mmccune. Thank you for putting this together!

Reply:
I'm a fan of using `cron.d` because it's a bit more flexible to modify.

Another note: you can use `install -D` and leave out the `install -d cron.monthly` above.